### PR TITLE
Add Clojure

### LINKED ISSUE #1147: Request for support Clojure
Please support the syntax highlight for [Clojure](https://clojure.org/) language. Thank you.

### PR DESCRIPTION
**Pull request by troglotit**
Closes #1147 

Copied definitions from https://github.com/jeluard/prism-clojure, example copied from https://learnxinyminutes.com/docs/clojure/

Tests are not so thorough, but they pass, and syntax highlighting looks decent.

### CODE REVIEW

File: components/prism-clojure.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        lookbehind: true
+    },
+    'boolean': /\b(true|false)\b/g,
+    'number': /\b-?(0x)?\d*\.?\d+\b/g,
```
Comment (Golmote) [inline components/prism-clojure.js]: Please make the parentheses non-capturing here too.
Also, `0x` usually indicates support for hexadecimal numbers. But here, letters `a-f` are not allowed. Is it intended?

File: examples/prism-clojure.html [hunk 2]
```diff
@@ -0,0 +1,386 @@
+<h2>Full example</h2>
```
Comment (Golmote) [inline examples/prism-clojure.html]: Please add something like this at the very beginning of this file:
```html
<h1>Clojure</h1>
<p>To use this language, use the class "language-clojure".</p>
```

File: tests/languages/clojure/comment_feature.test [hunk 3]
```diff
@@ -0,0 +1,15 @@
+;; this is comment
+; this is another comment
+; this is another comment
```
Comment (Golmote) [inline tests/languages/clojure/comment_feature.test]: Those two are exactly the same, aren't they? There is no need to test for the exact same thing twice.

File: tests/languages/clojure/keyword_feature.test [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+(do)
+(let)
+(var)
+(fn)
+
+----------------------------------------------------
+
+[
+	["punctuation", "("],
+  ["keyword", "def"],
```
Comment (Golmote) [inline tests/languages/clojure/keyword_feature.test]: Can you fix the indentation in this file, please?

**Comment by astoilkov** [comment]
@troglotit Are you willing to finish this pull request. It would be awesome Prism to have Clojure support.

**Comment by troglotit** [comment]
@astoilkov I had a wish of finishing it, but I was quite busy. I'd gladly hand this PR over if you wish to work on it, so open PR with pinging this PR, so I'd close this one.

**Comment by tkjone** [comment]
@astoilkov Do we know if any progress has been made on this?

**Comment by astoilkov** [comment]
@tkjone I haven't started working on this. But anyone with a little free time could fix the issues super quick. @Golmote did an amazing job of pointing out everything that should be done.

**Comment by tkjone** [comment]
Requested updates resolve in #1311

**Comment by Golmote** [comment]
#1311 is now merged, so I'm closing this PR.

**Pull request closed without merging**